sa/proto/sa.proto

```diff
@@ -179,6 +177,7 @@ message CountFQDNSetsRequest {
   reserved 1; // Previously windowNS
   repeated string dnsNames = 2;
   google.protobuf.Duration window = 3;
+  int64 limit = 4;
```

Comment:
Do you want this to be here? I believe this is in service to your other PR, so maybe it makes sense to land it early for the sake of deployability, but want to double-check that its inclusion in this PR is purposeful.

Reply:
Nope, thanks for catching 🤦🏻